src/core/frontend/emu_window.cpp

```diff
@@ -87,6 +51,19 @@ static bool IsWithinTouchscreen(const Layout::FramebufferLayout& layout, unsigne
     }
 }
 
+std::pair<f32, f32> EmuWindow::MapToTouchScreen(u32 framebuffer_x, u32 framebuffer_y) const {
+    std::tie(framebuffer_x, framebuffer_y) = ClipToTouchScreen(framebuffer_x, framebuffer_y);
+    /* const float x =
+        static_cast<float>(framebuffer_x - framebuffer_layout.screen.left) /
+        static_cast<float>(framebuffer_layout.screen.right - framebuffer_layout.screen.left);
+    const float y =
+        static_cast<float>(framebuffer_y - framebuffer_layout.screen.top) /
+        static_cast<float>(framebuffer_layout.screen.bottom - framebuffer_layout.screen.top);*/
+
+    // return std::make_pair(x, y);
+    return std::make_pair(0, 0);
```

Comment:
```suggestion
    const float x =
        static_cast<float>(framebuffer_x - framebuffer_layout.bottom_screen.left) /
                    static_cast<float>(framebuffer_layout.bottom_screen.right -
                                       framebuffer_layout.bottom_screen.left);
    const float y =
        static_cast<float>(framebuffer_y - framebuffer_layout.bottom_screen.top) /
                    static_cast<float>(framebuffer_layout.bottom_screen.bottom -
                                       framebuffer_layout.bottom_screen.top);

     return std::make_pair(x, y);
```
The code is mostly the same. But you refer to the bottom screen :P